sdk/textanalytics/Azure.AI.TextAnalytics/samples/README.md

```diff
@@ -11,26 +11,30 @@ description: Samples for the Azure.AI.TextAnalytics client library
 ---
 
 # Azure Text Analytics client SDK Samples
-Azure Cognitive Services Text Analytics is a cloud service that provides advanced natural language processing over raw text, and includes the following main features: 
-* Language Detection
-* Sentiment Analysis
-* Key Phrase Extraction
-* Entity Recognition (Named, Linked, and Personally Identifiable Information (PII) entities)
-* Healthcare Recognition
-* Extractive Text Summarization
-* Custom Entity Recognition
-* Custom Single and Multi Category Classification
+Azure Cognitive Services Text Analytics is one of the three services that are part of the Azure Cognitive Service for Language. This client library includes the following main features:
+
+* Extract Information: Use Natural Language Understanding (NLU) to extract information from unstructured text. For example:
+  * Key Phrase Extraction
+  * Entity Recognition (Named, Linked, and Personally Identifiable Information (PII) entities)
+  * Healthcare Entities Recognition
+  * Extractive Text Summarization
+  * Custom Named Entity Recognition
+* Classify Text: Use Natural Language Understanding (NLU) to detect the language or classify the sentiment of text you have
```

Comment:
* Classify Text: Use NLU to detect the language, analyze sentiment, or classify the text you have

Reply:
thank you!